lib/processLayerStyles.js

```diff
@@ -72,12 +75,62 @@ const layerFill = (layerStyle) => {
 
         }
 
+        if (fill.fillType == 4 && fill.isEnabled == true) { // Images
+          // Copy the image from the sketch file to the output folder
+          let imageOutputFolder = outputFolder + "/assets/Assets.xcassets"
+          let imageName = fill.image._ref.split("/").splice(-1)[0]
+          const name = imageName.split(".")[0]
+
+          // Note - this will be async, but we just carry on anyway because we don't use any of the output.
+          fs.mkdirSync(imageOutputFolder, { recursive: true })
+          sketchFile.contentsPath().then(contentsPath => {
+            // Create the asset bundle    
+            prepareAssetBundle(imageName, imageOutputFolder)
+            // Awful, but works around a bug that appears to be in `sketchFile.contentsPath()` where we try to read the 
+            // contents of the folder but it is empty, resulting in a ENOENT.
+            setTimeout(() => {
+              fs.copyFileSync(contentsPath + "/" + fill.image._ref, imageOutputFolder + `/${name}.imageset/` + imageName)
+            }, 1000)
+          })
+
+          // 0: do nothing
+          // 1: fill
+          // 2: stretch
+          // 3: fit
+          switch (fill.patternFillType) {
+            case 0:
+              backgroundArray.push(`Image("${name}", bundle: nil)`);
+              break;
+            case 1:
+              backgroundArray.push(`Image("${name}", bundle: nil).resizable().aspectRatio(contentMode: .fill)`);
+              break;
+            case 2:
+              backgroundArray.push(`Image("${name}", bundle: nil).resizable()`);
+              break;
+            case 3:
+              backgroundArray.push(`Image("${name}", bundle: nil).resizable().aspectRatio(contentMode: .fit)`);
+              break;
+            default:
+              backgroundArray.push(`Image("${name}", bundle: nil).resizable()`);
+              break;
+          }
+        }
       }
       return backgroundArray
     }
     // return "Color.clear"
 }
 
+const prepareAssetBundle = (imageName, assetBundleFolder) => {
+  const extension = imageName.split(".")[1]
+  const name = imageName.split(".")[0]
+  if (extension === "pdf") {
+    prepareAssetBundleForPDF(assetBundleFolder, name)
+  } else if (extension === "png") {
+    prepareAssetBundleForPNG(assetBundleFolder, name)
+  }
```

Comment:
Should we have an `else { console.log("Unknown file extension " + extension) }` ?